index.html

```diff
@@ -104,67 +106,80 @@ <h2 class="offer__header">Tittle 3</h2>
                 </div>
-                <div class="offer2__left">
-                    <h2 class="offer2__header">It tastes like Premium.
+                <div class="premium__left">
+                    <h2 class="premium__header">It tastes like Premium.
                     </h2>
-                    <h3 class="offer2__description">Better than an apple.
+                    <h3 class="premium__description">Better than an apple.
                     </h3>
-                    <p class="offer2__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+                    <p class="premium__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
                         tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
                         exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
                         reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
                         occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
                     </p>
-                    <button class="offer2__button" alt="przycisk subscribe now">
+                    <button class="premium__button" alt="subscribe now button">
                         Subscribe now
                     </button>
                 </div>
             </div>
         </div>
     </section>
 
-    <section class="offer3">
+    <section class="testimonials">
         <div class="container">
-            <div class="offer3__content">
-                <div class="offer3__up">
-                    <h2 class=" offer3__header">What Our Customers Say
+            <div class="testimonials__content">
+                <div class="testimonials__up">
+                    <div class="testimonials__up--header">
+                    <h2 class=" testimonials__header">What Our Customers Say
                     </h2>
-                    <img src="./src/assets/icon/bigQuotes.svg" class="offer3__quotes" alt="ikona cudzysłów" />
-                    <div class="offer3__table">
-                        <div class="offer3__collumn">
-                            <img src="./src/assets/icon/collumnLeft-offer3.svg" class="offer3__left"
-                                alt="ikona niebieskiej kolumny" />
-                            <img src="./src/assets/icon/littleQuoteW.svg" class="offer3__littleQuoteW"
-                                alt="ikona biała cudzysłów" />
-                            <h3 class="offer3__description1">“I love oranges!! tasty mmmmm…..”</h3>
-                            <p class="offer3__name1">Marjon Siero / Ironhack</p>
+                    <img src="./src/assets/icon/bigQuotes.svg" class="testimonials__quotes"
+                        alt="icon of big light-pink quotes" />
+                    </div>

```

Comment:
this should appear inside an element which is currently  `.testimonials__table`

Reply:
![image](https://user-images.githubusercontent.com/48392975/156885411-39996d40-8c9e-490f-8478-d71641d02f92.png)
 this all should be in .testimonials__table?